ethcore/src/state/account.rs

```diff
@@ -315,6 +315,11 @@ impl Account {
 		self.code_hash == SHA3_EMPTY
 	}
 
+	/// Check if account is basic (Has no code).
+	pub fn is_basic(&self) -> bool {
+		self.code_hash == SHA3_EMPTY
```

Comment:
i guess it's fine - this particular function should work at least (though at one point `new_contract` actually gave a non-basic account which just happened to have no code yet, but looks like it's now just an uncached basic account).